src/main/java/com/everis/boundary/CarResources.java

```diff
@@ -12,18 +10,27 @@
 import com.everis.control.CarService;
 import com.everis.entity.Car;
 
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+
 
 
 @Path("cars")
 @Consumes(MediaType.APPLICATION_JSON)
 @Produces(MediaType.APPLICATION_JSON)
+@OpenAPIDefinition(info = @Info(title = "Everis-Car-App_API", version = "0.0", description = "Car's CRUD Functionality"))
```

Comment:
So as to keep the Resource class cleaner, could you create an Interface and put the swagger annotation in this interface ?